src/components/activities/ActivityCard.jsx

```diff
@@ -99,9 +99,10 @@ class ActivityCard extends Component {
 
   constructor(props) {
     super(props);
+    const getStatuses = Object.keys(STATUSES).map(key => (STATUSES[key]));
```

Comment:
Why not just call it `statuses` and avoid defining a value when setting state?

Reply:
@kimani-ndegwa done